base_classes/NXpositioner.nxdl.xml

```diff
@@ -21,112 +21,75 @@
 #
 # For further information, see http://www.nexusformat.org
 -->
-<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXpositioner" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+<definition 
+    name="NXpositioner" 
+	type="group" 
+	extends="NXcomponent" 
+	category="base"
+	xmlns="http://definition.nexusformat.org/nxdl/3.1"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
+    >
     <doc>
-         A generic positioner such as a motor or piezo-electric transducer.
+        A generic positioner such as a motor or piezo-electric transducer.  
     </doc>
     <field name="name">
-        <doc>
-             symbolic or mnemonic name (one word)
-        </doc>
+        <doc>symbolic or mnemonic name (one word)</doc>
```

Comment:
I saw it here and in several other places, we should make an issue that is of kind quality improvements/formatting where we assure to always start docstrings always with a capital letter, symbols as well.